spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -16,6 +16,13 @@
     assert_response :redirect, :action => 'index'
   end
 
+  it "test_new" do
```

Comment:
Is there a test that can be done on the new save without required fields or is that already covered?